src/main.rs

```diff
@@ -1,14 +1,14 @@
+use core::panic;
 use std::io::{Stdin, Stdout, Write};
 
-fn main() {
+fn main() -> Result<(), std::io::Error> {
```

Comment:
Seria nice se importassemos o `std::io::Result` pra simplificar:

```suggestion
use std::io::Result;

fn main() -> Result<()> {
```

Reply:
Só uma explicação breve:
`std::io::Result` é como se fosse uma result que você não precisa especificar o erro, pois ele sempre vai ser `std::io::Error`.

Reply:
https://doc.rust-lang.org/stable/std/io/type.Result.html

Reply:
Boaa entendi feito